Lib/test/test_msvcrt.py

```diff
@@ -1,9 +1,8 @@
+import ctypes
```

Comment:
ctypes is not always available. Please use ``try: import ctypes except ImportError: ctypes = None``, and then skip tests using ctypes if ctypes is None.